### DISCUSSION TITLE
How can i contribute for translation?

### DISCUSSION THREAD

**Discussion by umtdemr** [mirumee/ariadne#676] [Q&A] [answered]
I want to contribute with translating ariadne docs to Turkish which is my native language. Is that possible?

**Comment by rafalp** [comment, marked as answer]
Not at the moment, I'm afraid. We would need to see if current docs setup supports i18n (we are using Docusaurus v1), and also make decision on the workflow. There's also question on reviewing those changes. Not speaking Turkish ourselves we won't be able to verify that PRs are valid.

**Reply by rafalp** [reply]
@umtdemr but as long as you reference back to `ariadnegraphql.org`, you can host Turkish translation somewhere yourself. :)
